placeholder.go

```diff
@@ -60,6 +64,36 @@ func (_ dollarFormat) ReplacePlaceholders(sql string) (string, error) {
 	return buf.String(), nil
 }
 
+type colonFormat struct{}
+
+func (_ colonFormat) ReplacePlaceholders(sql string) (string, error) {
```

Comment:
Can this be deduped with dollarFormat with another func, e.g. `replacePositionalPlaceholders(sql, prefix string)`?